sdk/python/foundation-models/system/inference/automatic-speech-recognition/asr-online-endpoint.ipynb

```diff
@@ -0,0 +1,238 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Automatic Speech Recognition Inference using Online Endpoints\n",
+    "\n",
+    "This sample shows how to deploy `automatic-speech-recognition` type models to an online endpoint for inference.\n",
+    "\n",
+    "### Task\n",
+    "`automatic-speech-recognition` (ASR) converts a speech signal to text, mapping a sequence of audio inputs to text outputs. Virtual assistants like Siri and Alexa use ASR models to help users everyday, and there are many other useful user-facing applications like live captioning and note-taking during meetings.\n",
+    "\n",
+    "### Model\n",
+    "Models that can perform the `automatic-speech-recognition` task are tagged with `task: automatic-speech-recognition`. We will use the `openai-whisper-large` model in this notebook. If you opened this notebook from a specific model card, remember to replace the specific model name. If you don't find a model that suits your scenario or domain, you can discover and [import models from HuggingFace hub](../../import/import-model-from-huggingface.ipynb) and then use them for inference. \n",
```

Comment:
The import links are broken on all the notebooks. Can you update all to (../../import/import_model_into_registry.ipynb)

Reply:
Fixed